src/main/java/com/gluonhq/substrate/target/PosixTargetConfiguration.java

```diff
@@ -38,6 +42,10 @@
 
     PosixTargetConfiguration(ProcessPaths paths, InternalProjectConfiguration configuration) {
         super(paths, configuration);
+
+        if (new Triplet(Constants.Profile.MACOS).equals(Triplet.fromCurrentOS())) {
```

Comment:
why is this in PosixTargetConfiguration if it is only relevant to MacOS?

Reply:
This applies to MacOS and iOS

Reply:
that is Darwin, not posix

Reply:
However, since this is a check on host and not on target, this does not belong in a TargetConfiguration at all (but probably in SubstrateDispatcher)